cmd/minikube/cmd/start.go

```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```

Comment:
@budhrg any reason why you moved the log statement. You put it at a place where it potentially will be printed several times. I am inclined to move it back.